src/JuMP.jl

```diff
@@ -215,6 +217,16 @@ for f in MathProgBase.SolverInterface.methods_by_tag[:rewrap]
     end
     eval(Expr(:export,f))
 end
+import MathProgBase.getobjbound
+function getobjbound(m::Model)
```

Comment:
It's inconsistent to confuse the solver's ``getobjbound`` and JuMP's ``getobjbound`` here when we have separate names for ``MathProgBase.getobjval`` and ``JuMP.getobjectivevalue``. We should leave ``getobjbound`` alone and implement a new ``JuMP.getobjectivebound``.